tl/candidate_generation/es_search.py

```diff
@@ -113,6 +113,95 @@ def create_fuzzy_query(self, search_term: str, size: int, properties):
 
         return query
 
+    def create_fuzzy_augmented_query(self,search_term: str, size: int, properties: List[str]):
```

Comment:
merge the functions `create_fuzzy_augmented_query` and `create_fuzzy_augmented_keyword_lower_query`, with an input parameters to specify case insensitive query